skopt/tests/test_forest_opt.py

```diff
@@ -56,9 +56,9 @@ def test_tree_based_minimize(name, minimizer):
     # the minimum, since for a decision tree any point greater than
     # max(sampled_points) would give a constant value.
     X0 = [[-5.6], [-5.8], [5.8], [5.6]]
-    check_minimize(minimizer, bench2, -5,
+    check_minimize(minimizer, bench2, -4.7,
```

Comment:
Was this breaking the test with the old numbers?

Reply:
(just want to know why these particular numbers used here and below)

Reply:
yes the tests were breaking. Makes me a bit uncomfortable because it isn't clear if these changes made the code more or less correct ... :-/

Reply:
Hm
Results with small number of n_calls for checking the procedure are normally quite noisy, so this might break simply by chance from time to time
There are a few options I see here:
1. Run the minimization procedure for some large number of repetitions, say 50, and then compare average of minimum objectives to some "normal" average min. This way results are more stable. To obtain such "normal" value, run the procedure with current minimizers, get avg min value, and eg add 0.1. Is computationally expensive - testing might time out.
2. Remove these tests. It might happen that some change leads to some large improvement in performance with large n_calls, but at a cost of a relatively small decrease in performance with smaller call budget, which would however breaks tests. 

I am leaning more towards option 1, but in principle fine with both options. For future: maybe there is an option to run slower tests somehow on a different infrastructure, where we could run such benchmarks regularly.

Reply:
I'd like to keep the tests even if they aren't that meaningful. We haven't had to adjust them for quite a while and when we do they serve more as a reason to discuss things. Overall not optimal but IMHO better than having tests that take a long time to run.

If we could run (a subset) of the benchmarks you put together once a day that would be awesome. Travis has a cron feature but I wonder if the resources we get through that are enough or maybe we need to create a set of tests for "once a day travis" runs. New issue?

Reply:
Yeah it is also ok to leave it like that. A better benchmarking is indeed a different issue.